### DISCUSSION TITLE
Cursor height setting

### DISCUSSION THREAD

**Discussion by Libsod** [wezterm/wezterm#4884] [General]
I recently upgraded to a new version of wezterm (not nightly). And my cursor height became smaller than line height.

I think you should definitely add a cursor_height setting. If there is something like this already, please tell me how I can make cursor height the same as line height, because I don't like the way it looks at all.

New cursor height:
<img width="231" alt="image" src="https://github.com/wez/wezterm/assets/90096350/f5cd3397-50f6-4f79-8b10-7ee45868da3d">

Old cursor height:
<img width="338" alt="image" src="https://github.com/wez/wezterm/assets/90096350/ad22d849-0ed6-422d-8974-6c3b4a96c98f">

My WezTerm version: 20240127-113634-bbcac864

**Comment by olimorris** [comment]
Yeah to add this would be a nice enhancement. Looks odd to have the cursor height < line height.

Also the changes to fonts on MacOS in the latest build have left me unable to bolden text.

**Comment by bew** [comment]
Idea of config interface:
- percentage like "100%" would fill line height
- font-specific size like "1em" for the current same-height-as-font

Are there other cases?
(wondering what should happen if using `2em`, leading to taller cursor than line height, maybe crop to line height?)

**Comment by Libsod** [comment]
Check - https://github.com/wez/wezterm/issues/2882#issuecomment-1925418509
